checkov/common/proxy/proxy_client.py

```diff
@@ -31,3 +31,13 @@ def send_request(self, request: requests.Request) -> requests.Response:
 def call_http_request_with_proxy(request: requests.Request) -> Any:
     proxy_client = ProxyClient()
     return proxy_client.send_request(request=request)
+
+
+def get_proxy_envs() -> Mapping[str, str] | None:
+    if os.getenv('PROXY_URL'):
+        proxy_env = os.environ.copy()
+        proxy_env["GIT_SSL_CAINFO"] = os.getenv('PROXY_CA_PATH')  # Path to the CA cert
+        proxy_env["http_proxy"] = os.getenv('PROXY_URL')  # Proxy URL
+        proxy_env["https_proxy"] = os.getenv('PROXY_URL')  # HTTPS Proxy URL (if needed)
+        return proxy_env
+    return None
```

Comment:
wouldn't the default return value be the original env vars?

Reply:
yes its can be. did None to not change the input to the clon_from function